src/current/v23.1/migration-overview.md

```diff
@@ -5,89 +5,351 @@ toc: true
+
+For an overview of lift-and-shift migrations to CockroachDB, see [Lift and Shift](#lift-and-shift). {% comment %}For considerations and details about the pros and cons of this approach, see [Migration Strategy: Lift and Shift](migration-strategy-lift-and-shift).{% endcomment %}
+
+#### Minimal downtime
+
+If your application cannot tolerate downtime, then you should aim for a "zero-downtime" approach. "Zero" means that downtime is reduced to either an absolute minimum or zero, such that users do not notice the migration.
+
+The minimum possible downtime depends on whether you can tolerate inconsistency in the migrated data:
+
+- *Consistent* migrations reduce downtime to an absolute minimum (i.e., from 30 seconds to sub-seconds) while keeping data synchronized between the source database and CockroachDB. **Consistency requires downtime.** In this approach, downtime occurs right before [cutover](#cutover-strategy), as you drain the remaining transactions from the source database to CockroachDB.
+
+- *Inconsistent* migrations can reduce downtime to zero. These require the most preparation, and typically allow read/write traffic to both databases for at least a small amount of time, thereby sacrificing consistency for availability. {% comment %}You can use the CockroachDB Live Migration Service (MOLT LMS) to run application queries simultaneously on your source database and CockroachDB.{% endcomment %} Without stopping application traffic, you perform an immediate [cutover](#cutover-strategy), while assuming that some writes will not be replicated to CockroachDB. You may want to manually reconcile these data inconsistencies after switching over.
+
+For an overview of zero-downtime migrations to CockroachDB, see [Zero Downtime](#zero-downtime). {% comment %}For details, see [Migration Strategy: Zero Downtime](migration-strategy-zero-downtime).{% endcomment %}
+
+### Cutover strategy
+
+*Cutover* is the process of switching application traffic from the source database to CockroachDB. Consider the following:
+
+- Will you perform the cutover all at once, or incrementally (e.g., by a subset of users, workloads, or tables)?
+
+	- Switching all at once generally follows a [downtime window](#downtime-window) approach. Once the data is migrated to CockroachDB, you "flip the switch" to route application traffic to the new database, thus ending downtime.
+
+	- Migrations with [zero or near-zero downtime](#minimal-downtime) can switch either all at once or incrementally, since writes are being synchronously replicated and the system can be gradually migrated as you [validate the queries](#validate-queries).
+
+- Will you have a fallback plan that allows you to reverse ("roll back") the migration from CockroachDB to the source database? A fallback plan enables you to fix any issues or inconsistencies that you encounter during or after cutover, then retry the migration.
```

Comment:
I'm using it here because we also see "roll back" used in the context of reverting a cluster upgrade, and this usage is currently more prevalent in our docs. In this case, "roll back" also has the connotation of switching directions from target back to source, so I think it's worth spelling out to capture that nuance.